src/Sylius/Behat/Service/Accessor/NotificationAccessor.php

```diff
@@ -81,7 +81,7 @@ public function isSuccessfullyUpdatedFor($resourceName)
      */
     public function isSuccessfullyDeletedFor($resourceName)
     {
-        return $this->hasMessage(sprintf('Success %s has been successfully deleted.', ucfirst($resourceName)));
+        return $this->hasMessage(sprintf('Success %s has been successfully deleted.', $this->convertToResourceName($resourceName)));
```

Comment:
That is weird. The argument is named `$resourceName`. Then we convert it to... resource name! :D